app/globals.css

```
@@ -1443,6 +1443,23 @@
    ----------------------------------------------------------------- */
 
+/* 리드 헤드라인 — 슬롯 네 곳이 제각각(clamp 2 / 2 / 3 / 6)이던 걸 한 곳으로 모은다.
+   2줄에서 자르고 넘치면 …. 줄 수를 바꾸려면 `--lede-headline-lines` 한 줄만 고치면 되고,
+   슬롯을 새로 만들 때도 이 클래스만 붙이면 나머지와 저절로 같아진다.
+
+   예약 높이(min-height)는 두지 않는다. footer가 `mt-auto`로 바닥에 고정돼 있어
+   헤드라인이 1줄이든 2줄이든 응원 버튼이 안 움직이고, 남는 높이는 body가 흡수한다 —
+   "몇 줄을 미리 잡아둘까"를 정할 필요 자체가 없어진다. */
+.lede-headline {
+  --lede-headline-lines: 2;
+  display: -webkit-box;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)